framework-for-proposals.md

```diff
@@ -0,0 +1,62 @@
+# Framework for proposals
+
+*This is an early draft and currently it does NOT represent a rough consensus,
+please use the issue tracker to iteratively improve it*
+
+
+## New suggestion
+
+Must include a problem statement and use cases. At
+this point, all that is sought is expressions of shared interest
+or questions of clarification. Commenters must not, at this stage:
+* Critique the problem statement or use cases (e.g., saying "this is not related to Solid")
+* Discourage further expressions of interest. (e.g., "It's too soon for this discussion.")
+
+The proponent may ask the Chairs to put the topic on the CG's
+meeting agenda for an introduction, to gauge interest, invite
+collaborators, and ask clarifying questions. The Chairs add the
+topic to the agenda at their discretion, or they may ask for the
+proponents to do additional work before putting it on the agenda.
+
+
+## Requirements
+
+After detecting shared interest (e.g., some number
+of people from different organizations say "I'm interested in
+this" on the GitHub discussion), the proponent and any
```

Comment:
Unless the intent is to use the GitHub Discussion feature, at the exclusion of GitHub Issues and/or PRs...
```suggestion
this" on the GitHub thread), the proponent and any
```